pkg/revocation/db.go

```diff
@@ -112,7 +112,9 @@ func (db *DB) List(ctx context.Context) (revs []*extensions.Revocation, err erro
 	if err != nil {
 		return nil, extensions.ErrRevocationDB.Wrap(err)
 	}
-
+	if len(keys) == 0 {
```

Comment:
Why is this case here?

All of the following should work when there aren't any keys. If it doesn't then it's probably a bug in the store implementation rather than a mistake here.

Reply:
As far as I understood, the redis client returns an error in this case.

Reply:
mget errors when you pass in an empty slice https://github.com/NodeRedis/node_redis/issues/125

Reply:
Generally yes we should try to homogenize the different implementation rather than push the concern to the user of the interface. The main reason is that otherwise the abstraction gets leaky, and we would end up having these "exceptional cases" for every caller and every implementation.

Reply:
I'll leave a note then that this happens

Reply:
I usually create a separate PR that fixes rather than leaving a note. Otherwise we'll end up collecting so many of those todo notes :D